larky/src/main/java/com/verygood/security/larky/modules/crypto/CryptoHashModule.java

```diff
@@ -80,6 +77,14 @@ public LarkyLongDigest SHA512(String truncate) throws EvalException {
     return new LarkyLongDigest(digest);
   }
 
+  @StarlarkMethod(name = "Keccak_256", parameters = {
+          @Param(name = "data", allowedTypes = {@ParamType(type = StarlarkBytes.class)})
+  }, useStarlarkThread = true)
+  public StarlarkBytes Keccak_256(StarlarkBytes data, StarlarkThread thread) {
```

Comment:
This can actually be very similar to the `SHA512` method even though Keccak comes from another package.

IMO, this is still very doable as we already have a `LarkyKeccakDigest` class: https://github.com/verygoodsecurity/starlarky/blob/5dbeadf79d22d66af02290487e9465f2f57a1b7a/larky/src/main/java/com/verygood/security/larky/modules/crypto/Hash/LarkyKeccakDigest.java


`BCMessageDigest` extends `java.security.MessageDigest` but includes a `Digest` interface: https://github.com/bcgit/bc-java/blob/f9cfd83a947f065dec3f4d997bb9a6373f9e9e25/prov/src/main/java/org/bouncycastle/jcajce/provider/digest/BCMessageDigest.java#L11

We know that `KeccakDigest` already implements `ExtendedDigest` https://github.com/bcgit/bc-java/blob/f9cfd83a947f065dec3f4d997bb9a6373f9e9e25/core/src/main/java/org/bouncycastle/crypto/digests/KeccakDigest.java#L13

So why isn't this just: 
```java
return LarkyKeccakDigest(new KeccakDigest(256));
```
?